docs/preferences.md

```diff
@@ -41,6 +41,24 @@ To switch versions:
 1. Select the version you want to change to.
 1. On the confirmation window, click **OK** to proceed.
 
+### Containter Runtime
+
+Set the [container runtime] for Rancher Desktop. Users have the option of [containerd] which provides namespaces for containers and the use of nerdctl or [dockerd (moby)] which enables the Docker API and the use of the Docker CLI. Only one container runtime will work function at a time.
```

Comment:
Minor change to delete the duplicate verbs - function, work

_Only one container runtime will function at a time._

Reply:
Fixed